tests/cupy_tests/functional_tests/test_vectorize.py

```diff
@@ -624,20 +622,17 @@ def my_func(x1, x2):
 
 class TestVectorize(unittest.TestCase):
 
-    @testing.for_all_dtypes(no_bool=True)
-    @testing.numpy_cupy_allclose(
-        rtol={'default': 1e-5,
-              numpy.float16: 1e-3 if runtime.is_hip else 1e-5})
-    def test_vectorize_arithmetic_ops(self, xp, dtype):
+    @testing.numpy_cupy_allclose(rtol=1e-5)
+    def test_vectorize_arithmetic_ops(self, xp):
         def my_func(x1, x2, x3):
             y = x1 + x2 * x3 ** x1
             x2 = y + x3 * x1
             return x1 + x2 + x3
 
         f = xp.vectorize(my_func)
-        x1 = testing.shaped_random((20, 30), xp, dtype, seed=1, scale=5)
-        x2 = testing.shaped_random((20, 30), xp, dtype, seed=2)
-        x3 = testing.shaped_random((20, 30), xp, dtype, seed=3)
+        x1 = testing.shaped_random((20, 30), xp, xp.float32, seed=1, scale=5)
```

Comment:
Question: why is this change needed?

Reply:
Because this test overflows on int8 inputs.

Reply:
I see, then how about shrinking the data size to, say, `(12, 16)`? As `TestVectorize` is a collection of realistic use-case tests, it makes sense to me to test dtype combination here.

Reply:
Fixed in a1c8eac.

Reply:
Thanks! (I just noticed I misread `shaped_random` as `shaped_arange` 😅 the fix looks good anyways)